README.md

```diff
@@ -106,6 +106,27 @@ It allows overriding the value, that is specified in the `cypress.json`
 cy.saveHar({ outDir: './hars' });
 ```
 
+Generate HAR file only for chrome, if it is not interactive run, and if test failed.
+```
+beforeEach( function () {
+    const isInteractive = Cypress.config('isInteractive');
+    const isChrome = Cypress.browser.name === 'chrome';
+
+    if (!isInteractive && isChrome) {
+        cy.recordHar();
+    }
+});
+
+afterEach( function () {
+    const { state } = this.currentTest;
+    const isInteractive = Cypress.config('isInteractive');
+    const isChrome = Cypress.browser.name === 'chrome';
+
+    if (!isInteractive && isChrome && state !== 'passed') {
+        cy.saveHar();
+    }
+});
```

Comment:
```suggestion
beforeEach(function () {
    const isInteractive = Cypress.config('isInteractive');
    const isChrome = Cypress.browser.name === 'chrome';

    if (!isInteractive && isChrome) {
        cy.recordHar();
    }
});

afterEach(function () {
    const { state } = this.currentTest;
    const isInteractive = Cypress.config('isInteractive');
    const isChrome = Cypress.browser.name === 'chrome';

    if (!isInteractive && isChrome && state !== 'passed') {
        cy.saveHar();
    }
});
```

Reply:
agree to the suggestion. Thanks for pointing out. Fix is just pushed.